lib/mix/tasks/appsignal.diagnose.ex

```diff
@@ -240,7 +240,11 @@ defmodule Mix.Tasks.Appsignal.Diagnose do
   defp print_configuration(config) do
     IO.puts("Configuration")
 
-    Enum.each(config[:options], fn {key, _} = option ->
+    # Filter out the diagnose_endpoint config option. Users don't need to see
+    # the config option. It's a private config option.
+    options = Enum.reject(config[:options], fn {key, _} -> key == :diagnose_endpoint end)
```

Comment:
Shouldn't this be called: `filtered_options`?

Reply:
Yes. Fixed.